### PR TITLE
protein-translation: sync

### PR DESCRIPTION
**Pull request by senekor**
[no important files changed]

### CODE REVIEW

File: rust-tooling/generate/templates/default_test_template.tera [hunk 1]
```diff
@@ -3,7 +3,7 @@ use crate_name::*;
 {% for test in cases %}
 #[test]
 #[ignore]
-fn {{ test.description | slugify | replace(from="-", to="_") }}() {
+fn {{ test.description | snake_case }}() {
```
Comment (senekor) [inline rust-tooling/generate/templates/default_test_template.tera:6]: piggyback

**Comment by senekor** [comment]
blocked on [this discussion](https://forum.exercism.org/t/incorrect-test-in-protein-translation/10616)

**Comment by senekor** [comment]
I struggled with the separation of the `name_for` and `of_rna` methods, which is not intended by problem-specifications. I considered to map all tests with only one output amino acid to the `name_for` method, but that doesn't work because of some edge cases:
- one amino acid and then immediately a stop codon (should test `of_rna`)
- invalid codons returning `None` (should test `name_for`)

So the solution I came up with for now is to add all previous `name_for` tests as custom tests. This leads to duplicated tests (same input once for `name_for` and once for `of_rna`) but preserves the existing exercise design.

**Review by ErikSchierboom — APPROVED** [review]
